content/en/blog/_posts/Free-Katacoda-Kubernetes-Tutorials-Are-Shutting-Down

```diff
@@ -0,0 +1,17 @@
+---
+layout: blog
+title: "Free Katacoda Kubernetes Tutorials Are Shutting Down"
+date: 2023-02-14
+slug: Free-Katacoda-Kubernetes-Tutorials-Are-Shutting-Down
+evergreen: true
+---
+
+**Author**: Natali Vlatko, SIG Docs Co-Chair for Kubernetes
```

Comment:
Feel free to mention that you are from Wayfair. Also, nit, I'd lowercase “co-chair”.

More like: Kubernetes SIG Docs co-chair

Reply:
Wayfair and I recently parted ways, hence only mentioning my k8s role 🙂